README.md

```diff
@@ -1,2 +1,2 @@
-# measure-api
-Micro-service for metric measurements and handling of scouts' requests.
+# measurement-api
+Micro-service for metric measurement and handling of scouts' requests.
```

Comment:
Any specific reason for calling it measurement and not measurements?
I feel measurements would be apt considering it deals with multiple scouts sending multiple measurements